wpilibcExamples/src/main/cpp/examples/SwerveDrivePoseEstimator/cpp/SwerveModule.cpp

```diff
@@ -64,13 +64,15 @@ void SwerveModule::SetDesiredState(
   const auto driveOutput = m_drivePIDController.Calculate(
       m_driveEncoder.GetRate(), state.speed.value());
 
-  const auto driveFeedforward = m_driveFeedforward.Calculate(state.speed);
+  const auto driveFeedforward =
+      m_driveFeedforward.Calculate(referenceState.speed, state.speed);
 
   // Calculate the turning motor output from the turning PID controller.
   const auto turnOutput = m_turningPIDController.Calculate(
       units::radian_t{m_turningEncoder.GetDistance()}, state.angle.Radians());
 
   const auto turnFeedforward = m_turnFeedforward.Calculate(
+      units::radians_per_second_t{m_turningEncoder.GetRate()},
```

Comment:
Don't use an encoder here, because this is a feedforward.

Reply:
I'll make this change to driveFeedforward as well